src/connection/passwordless/actions.js

```diff
@@ -45,9 +33,22 @@ function startPasswordlessErrorMessage(m, error, medium) {
   );
 }
 
+export function requestPasswordlessEmail(id) {
+  validateAndSubmit(id, ['email'], m => {
+    sendEmail(m, requestPasswordlessEmailSuccess, requestPasswordlessEmailError);
+  });
+}
+
+export function requestPasswordlessEmailSuccess(id) {
+  swap(updateEntity, 'lock', id, m => {
```

Comment:
what's `updateEntity` and where is it defined?

Reply:
it's the function that updates the model https://github.com/auth0/lock/blob/master/src/store/index.js#L26